Create transaction privacy page #175 (#349)
BitcoinDesign/Guide: guide/payments/introduction.md

---

### [Transaction privacy]({{ '/guide/payments/privacy/' | relative_url }})

> An overview of how to help users maintain their financial privacy while using Bitcoin.

Reviewer: This reads more like a general privacy overview wheras this page mostly talks about privacy in the context of transactions.

Author: How about this @Bosch-0 ?

"An overview of how to help users maintain their financial privacy while transacting with Bitcoin."

Reviewer 2: 'An overview of how to help users maintain transaction privacy with Bitcoin' ?
This commit suggestion is anyway dealing with the missing link, so go ahead and commit.

> An overview of how to help users maintain their financial privacy while using Bitcoin.

Reviewer 2: ```suggestion
An overview of how to help users maintain transaction privacy with Bitcoin.
```
